modules/stats.php

```diff
@@ -70,6 +73,33 @@ function stats_enqueue_dashboard_head() {
 }
 
 /**
+ * Checks do_not_track config.
+ *
+ * @return true if config honors DNT, false if not.
+ */
+function stats_honor_dnt_header_for_stats() {
+	$options = stats_get_options();
+
+	if ( true === $options['do_not_track'] )
```

Comment:
Could you use braces here, as per [the WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style)?